it/resources/scenario_1_transformed_request.json

```diff
@@ -1 +1 @@
,"name":{"firstName":"Albert","lastName":"Einstien","middleName":"G","title":"0001"},"position":"02","declaration":true,"overseas":false},"admin":{"acknowledgmentReference":"","credentialID":"/newauth/credentialId/scenario_1_request","sessionID":"sessionId","welshIndicator":false,"applicationDate":"T_O_D_A_Y"},"organisation":{"applicationType":"0","emailAddress":"abc@gmail.com","countryEstd":"1","orgName":"ABC","telephoneNumber":"07700 900 982","mobileNumber":"07700 000 111"},"addressDetails":{"differentCorrespondence":false,"officialAddress":{"addressLine1":"7","addressLine2":"Morrison street","nonUKCountry":"IN","nonUKAddress":true}}},"partner":[{"responsiblePerson":{"action":"1","relation":"2"},"individualDetails":{"mobilePhone":"07700 900 981","dayPhoneNumber":"07700 900 982","name":{"firstName":"Albert","lastName":"Einstien","middleName":"G","title":"0001"},"dateOfBirth":"2000-12-11","position":"02","nino":"QQ123456C"},"addressDetails":{"currentAddress":{"postcode":"G27JD","addressLine1":"2","addressLine2":"Dubai Main Road","addressLine3":"line3","addressLine4":"line4","nonUKAddress":false}},"type":"1","addPartner":{"effectiveDateOfChange":"T_O_D_A_Y"}},{"responsiblePerson":{"action":"1","relation":"1"},"individualDetails":{"mobilePhone":"07700 900 981","dayPhoneNumber":"07700 900 982","name":{"firstName":"David","lastName":"Beckham","title":"0001"},"dateOfBirth":"2000-12-11","position":"05","nino":"QQ123456C"},"addressDetails":{"currentAddress":{"addressLine1":"3","addressLine2":"Morrison Street","addressLine3":"Bill Tower","nonUKCountry":"IT","nonUKAddress":true}},"type":"1","addPartner":{"effectiveDateOfChange":"T_O_D_A_Y"}}],"charity":{"operationAndFunds":{"incomeReceivedToDate":121,"northernIreland":false,"otherAreaOperation":true,"futureFunds":"other, donations, tradingSubsidiaries, tradingIncome, fundraising, grants, membershipSubscriptions, investmentIncome","ukWide":false,"operationAndFundsCommon":{"financialAccounts":true,"accountPeriodEnd":"0101"},"englandAndWales":true,"estimatedGrossIncome":123,"scotland":false,"overseas":false},"charityOrganisation":{"nonRegReason":"1","registeredRegulator":false},"orgPurpose":{"charitablePurposes":{"other":false,"education":false,"animalWelfare":true,"humanRights":false,"healthOrSavingOfLives":false,"religion":false,"artsCultureOrScience":false,"citizenshipOrCommunityDevelopment":false,"amateurSport":true,"environmentalProtection":false,"reliefOfPoverty":false,"reliefOfYouthAge":false,"armedForcesOfTheCrown":false},"charitableObjectives":{"objectivesA":"qweqwewqesdfsdfdgxccvbcbre664354wfffgdfgdq34tggnchjn4w7q3bearvfxasxe14crtgvqweqwewqesdfsdfdgxccvbcbre66"},"publicBenefit":{"publicBenefitA":"qweqwewqesdfsdfdgxccvbcbre664354wfffgdfgdq34tggnchjn4w7q3bearvfxasxe14crtgvqweqwewqesdfsdfdgxccvbcbre66"}},"aboutOrganisation":{"governingApprovedDoc":false,"documentEnclosed":"1","aboutOrgCommon":{"otherDocument":"Other Documents for Charity","effectiveDate":"2014-07-01"},"governingApprovedWords":true}}}}
```

Comment:
@ryandac could we cleanse this data a bit further? There seem to be some names we could improve i.e not use comedy value names, nino could be AA000001A. For examples of test data in the open see https://confluence.tools.tax.service.gov.uk/display/CD/Test+Data+in+the+Open

Reply:
test data has been improved